src/scss/grommet-core/_objects.brick.scss

```diff
@@ -0,0 +1,94 @@
+.brick {
+  box-shadow: 1px 1px 5px #eee;
+  padding: $inuit-base-spacing-unit;
+  position: relative;
+  margin: 0 10px 10px 0;
+  float: left;
+  max-width: calc(100% - 10px);
+
+  .brick--label {
+    position: absolute;
+    top: 0;
+    right: 0;
+    left: 0;
+    bottom: 0;
+
+    span {
+      text-transform: uppercase;
+      text-decoration: none;
+      color: $text-color;
+      position: absolute;
+      left: $inuit-base-spacing-unit;
+      bottom: $inuit-base-spacing-unit;
+    }
+  }
+
+  &[class*="background-color-index-"] span {
+    color: #fff;
+  }
+}
+
+.brick--1-1 {
+  width: calc(25% - 10px);
+
+  &:after {
+    padding-top: 100%;
+    display: block;
+    content: '';
+  }
+
+  @media only screen and (max-width: 1000px) {
+    width: calc(50% - 10px);
+  }
+}
+
+.brick--1-2 {
+  width: calc(25% - 10px);
+
+  &:after {
+    padding-top: calc(200% + 58px);
```

Comment:
Updated the CSS to use the inuit spacing variable for all the `px` values in this file.